src/components/Button/index.tsx

```diff
@@ -12,16 +12,15 @@ const Base = styled(RebassButton)<
   {
     padding?: string
     width?: string
-    borderRadius?: string
+    $borderRadius?: string
```

Comment:
does `$` just change the property name so there's no conflict?

Reply:
no specifically it tells styled-components to not set the prop as a DOM property